.github/workflows/test.yml

```diff
@@ -29,6 +31,12 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      # Check if xfs config file exists in docker image
+      - name: Check XFS Config File
+        run: |
+          docker build -t xfs-test:latest -f dev/Dockerfile .
```

Comment:
To really be sure we should test against the production image, but I understand thats harder to do because the binaries are compiled outside of the image. As we also run the e2e tests with the non-prod image it does not make the situation worse than it was before :D

Reply:
I just realized we can just put two dummy files there. The main point is to test if file is present after installing xfsprogs via the package manager. No need for the actual binaries to be there.